flow/util/genRuleFile.py

```diff
@@ -290,6 +276,13 @@
     elif option['mode'] == 'abs_padding':
         rule_value = abs(metrics[field]) * (1 + option['padding'] / 100)
 
+    elif option['mode'] == 'metric':
+        rule_value = metrics[option['metric']] * option['padding'] / 100
+
+    if (field == 'cts__design__instance__count__setup_buffer'
+            or field == 'cts__design__instance__count__hold_buffer'):
+        rule_value = max(rule_value, metrics[field] * 1.1)
```

Comment:
I kept it because we might need to update the metrics before we fix the root cause. If I remove the code now, we might update the metrics and make it fail in the future.

Reply:
ok